command/apply.go

```diff
@@ -251,7 +251,7 @@ func (c *ApplyCommand) Run(args []string) int {
 	}
 
 	if !c.Destroy {
-		if outputs := outputsAsString(state, ctx.Module().Config().Outputs, true); outputs != "" {
+		if outputs := outputsAsString(state, []string{"root"}, ctx.Module().Config().Outputs, true); outputs != "" {
```

Comment:
Note for later: perhaps we should make `terraform.rootModulePath` an exported constant so we can reference from here.